### DISCUSSION TITLE
hidden md:flex does not work

### DISCUSSION THREAD

**Discussion by younixue3** [tailwindlabs/tailwindcss#4878] [Help] [answered]
Hello everyone, I'm a newbie at TailwindCss, I recently do Laravel Website with TailwindCss now I found an issue, I don't know how this happens, I do a class `<div class="hidden md:true">` but after I change to md resolution, the element still hidden

my code

`<nav class="hidden md:flex space-x-10">`

**Comment by olets** [comment]
I'm unable to replicate that. Does this minimal example work the way you expect? https://play.tailwindcss.com/gJgFbKuTC0

< md | ≥ md
---|---
![image](https://user-images.githubusercontent.com/3282350/124320355-72b30e00-db17-11eb-9184-4573e93f7716.png)|![image](https://user-images.githubusercontent.com/3282350/124320370-7cd50c80-db17-11eb-9435-d477b7b72cf5.png)

**Reply by younixue3** [reply]
yeah, but in my code that doesn't work, I code with Laravel 8, i follow the installation from tailwindcss documentation

**Reply by younixue3** [reply]
maybe conflict because in class hidden the style is `display:none;` meanwhile in class flex is `display:flex`

**Comment by younixue3** [comment, marked as answer]
my Experience, if want to use TailwindCss in Laravel, follow the documentation from Laravel Website, I don't know how this happend, but i do it, and done. **for Laravel user, don't follow from TailwindCss for the installation** on my experience

**Comment by lance1572** [comment]
I'm not sure if anyone tried this but for those that have this glitch. I checked tailwind play and it worked which baffled me. As an aside, my coworkers didn't have the issue on Chrome. So I kept digging in the dev tools and found the code below. After a little reddit dive I learned its an extension injecting the style which sure enough was overriding tailwind. For me it was the Show Window Size extension but yours maybe different. I disabled mine and  the tailwind classes started working. Give it check to see.  Hope this helps.

```
_injected stylesheet_

.hidden {
    visibility: hidden;
}
```
